### ISSUE TITLE
[Feature Request] Drawing 1D histograms in an XYFrame

### ISSUE THREAD

**Issue by sangaline** [nteract/semiotic#461]
I see that it's possible for Semiotic to automatically bin data and plot histograms as part of an ordinal summary, but I can't find anything relating to histograms in an `XYFrame` (other than the 2D ordinal summary plots). Is there anything analogous to Matplotlib's [pyplot.hist()](https://matplotlib.org/api/_as_gen/matplotlib.pyplot.hist.html) method for binning and drawing 1D histograms with two numerical axes? If not, any thoughts on the best way to accomplish this using the library?

**Comment by emeeks** [comment]
Could you post a screenshot or link to what the output would look like? I can't wrap my head around how a 2D histogram works if it's not point density/heatmap/hexbin kinds of things.

**Comment by sangaline** [comment]
The histogram itself is 1D, I just want to display the counts in each bin on the Y-axis. Here's a Matplotlib example:

![image](https://user-images.githubusercontent.com/8236564/56054269-6834b000-5d24-11e9-8996-ecc2e87b1425.png)

The closest thing I was able to find in the Semiotic examples was the histogram ordinal plot:


![image](https://user-images.githubusercontent.com/8236564/56056329-d62fa600-5d29-11e9-87bd-9a2a961fd3b3.png)

I was also able to hack together something similar in an `XYFrame` with an area chart, but I had to manually bin the data and also manage drawing the bin structure myself:

![image](https://user-images.githubusercontent.com/8236564/56056560-856c7d00-5d2a-11e9-9291-5012ff978be5.png)

It would similarly be possible to do things like add a visual separation between the bins, but this would require even more preprocessing of the data to generate the series of coordinates.

I haven't experimented with tooltips on this yet, but it looks like they would be associated with the bin edges rather than the bins themselves because that's where the data points are.
